mypyc/ir/rtypes.py

```diff
@@ -241,7 +242,10 @@ def __repr__(self) -> str:
                               ctype='int64_t')  # type: Final
 # integer alias
 c_int_rprimitive = int32_rprimitive
-c_pyssize_t_rprimitive = int64_rprimitive
+if sys.maxsize < (1 << 31):
```

Comment:
You can store the result of this check into a constant in `mypyc.common` so that it can be easily reused.